components/cam/bld/namelist_files/use_cases/2010_scream_hr_dyamond2.xml

```diff
@@ -151,7 +104,7 @@
 <drydep_list            >'H2O2', 'H2SO4', 'SO2'</drydep_list>
 
 <!-- sim_year used for CLM datasets and SST forcings -->
-<sim_year>2015</sim_year>
+<sim_year>2010</sim_year>
```

Comment:
As above, is this correct for DYAMOND?

Reply:
Yeah, this is what we'd been wanting to set (2010, rather than 2015).The IC file and CLM files are actually defined in the use_case files, so it actually won't have an impact.